src/File.php

```diff
@@ -34,6 +34,11 @@ public function collection(string $key): Collection
         return $collection;
     }
 
+    public function single(string $key):object{
+        $single = new Single($key, $this->writer);
+        return $single;
+    }
```

Comment:
We could take the both inputs `$key` and `$value` to this function so that it is easier for the library user.

Currently:

```php
$file->single('key')->addValue('value');
```

Better:

```php
$file->value('key', 'value');
```

Reply:
done

Reply:
Return type is wrong